docroot/themes/humsci/su_humsci_theme/patterns/molecules/featured-card/featured-card.ui_patterns.yml

```diff
@@ -0,0 +1,36 @@
+featured_card:
```

Comment:
Maybe we can rename this to something like "dated_vertical_card"

Reply:
Yeah, more specific here is probably good. That is, what separates this from a vertical card? I am going with date_stacked_vertical_card.